examples/run_lm_finetuning.py

```diff
@@ -120,13 +121,32 @@ def __getitem__(self, item):
         return torch.tensor(self.examples[item])
```

Comment:
^^ I did not add the last truncated example in the `TextDataset` (even though it should be handled by the padding and collation function below) @LysandreJik